### DISCUSSION TITLE
бонус Меча Альрика

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#71] [General]
Меч Альрика это единственный в Готике 2 классик меч дающий бонус к навыку. этот бонус нигде не прописан и меч по задумке ничем не отличается от обычного.

при получение бонуса от меча альрика, может поменять стойка с новичка на боец или с бойца на мастре. при этом при снятие меча стойка обратно не меняется.

**Comment by dosinabox** [comment]
А у самого Альрика меняется стойка при получении этого меча? А в НВ?

**Reply by Kor-Angar** [reply]
речь про героя у которого например навык 20%. с Мечём навык становится 30% и стойка меняется. 

***

В НВ такого бага нет, так как это процент который повышается/понижается фиксили миллион раз. так как оружия дающего бонус гора.

**Comment by Kor-Angar** [comment]
у Альрика в каих то версиях менялись параметры, то есть повышался навык после того как герой выйграет несколько дуэлей. есть ли это в классике или добавили в НВ я не знаю.

по моему у Альрика изначально навык < 30. а с с мечём должен быть 20+10=30 и соответственно произойти смена стойки.

если у Альрика скриптами меняются параметры и он становится более жёсткий, то и стойка у него поменяется, в таком случае не принципиально влияет меч на его стойку или нет.
